src/main/resources/static/js/script.js

```diff
@@ -0,0 +1,77 @@
+window.onload=console.log("ready");
+//window.onload=toggleItems();
+var parentForm = document.querySelector("#mainForm");
+parentForm.addEventListener("click", callRelevantFunction, false);
+
+// Calls the button function related to the clicked button.
+// The name is retrieved from the 'name' of the input field, so make sure that name and the function name is equal.
+function callRelevantFunction(e){
+	if(e.target !== e.currentTarget){
+		var clickedItem = e.target.name;
+		if (e.target.type == "button"){
+			window[clickedItem]();
+		}
+	}
+	e.stopPropagation();
+}
+
+// If this toggling functionality stays, clean everything up in 1 neat function
+function toggleItems(){
+	document.querySelector(".getReservation").style.visibility = "hidden"
```

Comment:
dit kun je in één keer doen, dat is ten slotte waar classes voor zijn. Geeft al bovenstaande elementen bijvoorbeeld de class .reservation, of .toggleable of verzin iets creatiefs